js/impact.js

```diff
@@ -2458,7 +2458,7 @@ var GLPIImpact = {
          html += "<ul>";
 
          elements.forEach(function(element) {
-            var link = "./" + url + ".form.php?id=" + element.id;
+            var link = CFG_GLPI.root_doc + "/front/" + url + ".form.php?id=" + element.id;
```

Comment:
This may broke plugins item, no ?

Reply:
This method is used to generate URLs for tickets/problems/changes and is called only from `buildOngoingDialogContent`.